FreeRTOS/Test/CBMC/proofs/MakefileCommon.json

```diff
@@ -12,7 +12,8 @@
 	"__PRETTY_FUNCTION__=__FUNCTION__",
          "CBMC",
 	"'configASSERT(X)=__CPROVER_assert(X,\"Assertion Error\")'",
-    "'configPRECONDITION(X)=__CPROVER_assume(X)'"
+    	"'configPRECONDITION(X)=__CPROVER_assume(X)'",
+	"'_static= '"
```

Comment:
No space?  Also _volatile and _inline?

Reply:
I mean consider omitting the space.

Reply:
I don't remeber correctly but I believe that I tried omitting the space and it broke...

Reply:
Done! Thanks! I removed the space and added `_volatile` in the `.json`s.
As far as inline is concerned, none of the TCP patches did remove inline. Therefore, we can add it later when required. Also, the macro shall be defined as `#define _inline portINLINE`.